app/views/shared/_invitations.haml

```diff
@@ -1,15 +1,9 @@
-= t('.share_this')
+= t(".share_this")
 = invite_link(current_user.invitation_code)
-#invitations-button.btn.btn-link{ "data-toggle" => "modal"}
+.invitations-link.btn.btn-link{"data-toggle" => "modal"}
```

Comment:
You could just add the id `invitations-button` here while keeping the `.invitations-link` class.